### ISSUE TITLE
[RFC] Change yarn to pnpm

### ISSUE THREAD

**Issue by simenandre** [bjerkio/create-bjerk-typescript#268]
Main driver for me is:
- ` pnpm config set auto-install-peers true` (Automatically install peer dependencies, no need to keep maintaining it in our own package.json)
- Less boilerplate in each repository (no `.yarn` folder)

As we're moving towards using tools like `corepack`, I no longer feel we need to pack the _binary_ for our package manager with our code.

Personally not sure about this yet, but I want to request some comments.

**Comment by mrboen94** [comment]
One thing to consider is the compability, pnpm is great and ive had mostly no issues with it. Sometimes it seems to break on M1 macs though (but usually resolvable). Its a fine line to balance between what is best in theory and what is best in practise. I'd argue pnpm should wait just a little longer to see if the bug tracking in their repository calms down a bit.

This might be a good change to look further into around July/August as it seems to move pretty fast right now.

TL;DR: I think its the right direction, but I would personally wait a few months.
